nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/pom.xml

```diff
@@ -214,5 +219,9 @@
             <version>1.17.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.xerial.snappy</groupId>
+            <artifactId>snappy-java</artifactId>
+        </dependency>
```

Comment:
Is it necessary to introduce the explicit dependency on `snappy-java`? Is it it no longer a transitive dependency of Avro?